### ISSUE TITLE
Правила

### ISSUE THREAD

**Issue by kswaldemar** [gamover321/raicup2017#1]
Согласно правилам russianaicup.ru, запрещено выкладывать исходники своего решения в открытый доступ, пока идет соревнование.
Если ваш код кто-то скопирует, то вы оба получите бан.

**Comment by sat2707** [comment]
Да, это так, поддержу предыдущего оратора. Советую сделать реп приватным :)
